lib/engine-dylib/src/artifact.rs

```diff
@@ -291,12 +291,32 @@ impl DylibArtifact {
             }
         };
 
+        // Set 'isysroot' clang flag if compiling to iOS target
+        let ios_sdk_flag = {
+            if target_triple.operating_system == OperatingSystem::Ios {
+                "-isysroot/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk"
+            } else {
+                ""
+            }
+        };
+
+        // Get the location of the 'ld' linker for clang
+        let fuse_linker = {
+            let ld_install = which::which("ld");
+            if ld_install.is_ok() {
+                ld_install.unwrap().into_os_string().into_string().unwrap()
+            } else {
+                "ld".to_string()
+            }
+        };
+
```

Comment:
Please configure and add this flag if and only if the `target_triple` is for `aarch64-apple-ios` :-).

Reply:
Is this better @Hywan?? We need to be able to find the linker for `x86_64-apple-ios` && `aarch64-apple-ios`.